subprojects/build-init/src/main/java/org/gradle/api/tasks/wrapper/Wrapper.java

```diff
@@ -357,10 +357,13 @@ public String getDistributionUrl() {
      *
      * <p>If {@link #getGradleVersion()} is not set, will return null.
      *
-     * <p>The wrapper downloads a certain distribution only once and caches it. If your distribution base is the
+     * <p>The wrapper downloads a certain distribution and caches it. If your distribution base is the
      * project, you might submit the distribution to your version control system. That way no download is necessary at
      * all. This might be in particular interesting, if you provide a custom gradle snapshot to the wrapper, because you
      * don't need to provide a download server then.
+     *
+     * After 30 days of no use, the wrapper has the possiblity of being deleted. If this behavior is not desired,
+     * Set the property `org.gradle.disable.cache.cleanup` to true in the gradle.properties file in your gradle home.
```

Comment:
nit-pick: I would lower-case `Set` as it isn't the start of a sentence, and will look weird in the generated Javadoc.